client/src/App.js

```diff
@@ -1,9 +1,24 @@
-import logo from './logo.svg';
-import './App.css';
+import { Route, Switch } from "react-router-dom";
+import Home from "./components/home";
+import RestaurantPage from "./components/restaurant";
+//import { useState } from "react";
+
+import "./App.css";
 
 function App() {
+  //Set up paths for Home Page and Restaurant Page content.
   return (
-    <h1>Yelpington</h1>
+    <div>
+      <header>
+        <a href="/">
+          <h1>NYC's Best Food and Drink</h1>
+        </a>
```

Comment:
Since we are using `react-router-dom` we may want to switch this `<a>` tag for a `<Link>` tag. This way we're giving that power to react still rather than controlling it yourself